content/app-dev/workflow/tracing.en.md

```diff
@@ -0,0 +1,61 @@
+```bash
+2022-07-11T16:09:08.742923Z  INFO request{method=GET path=/vets actor_id=MA5DZLFF733IR7TIDMBNOUMDS7I32NFUJIZ7LBSS5ED3V6GPFTDZJXZ3}: warp::filters::trace: processing request
+2022-07-11T16:09:08.765667Z DEBUG rpc:rpc:dispatch{public_key=MBZ3XLCME3RZBF7GQJ2LNIFZXHOGEJ2I7GVRHSIOPERWQXGMAH5NCI4F operation=SqlDb.Query}:query{actor_id=Some("MBZ3XLCME3RZBF7GQJ2LNIFZXHOGEJ2I7GVRHSIOPERWQXGMAH5NCI4F")}: sqldb_postgres: executing read query
+2022-07-11T16:09:08.784536Z  INFO request{method=GET path=/vets actor_id=MA5DZLFF733IR7TIDMBNOUMDS7I32NFUJIZ7LBSS5ED3V6GPFTDZJXZ3}: warp::filters::trace: finished processing with success status=200
+tempo_1      | level=info ts=2022-07-11T16:09:11.238063552Z caller=distributor.go:409 msg=received spanid=08329a3ab0d07420 traceid=3254bd41d3d557cb9f71fd3c3edb02fb
+tempo_1      | level=info ts=2022-07-11T16:09:11.23825326Z caller=distributor.go:409 msg=received spanid=a58d9cb2caf2611f traceid=3254bd41d3d557cb9f71fd3c3edb02fb
+...
+tempo_1      | level=info ts=2022-07-11T16:09:12.538655178Z caller=distributor.go:409 msg=received spanid=0ef65bc127f42a3e traceid=3254bd41d3d557cb9f71fd3c3edb02fb
+tempo_1      | level=info ts=2022-07-11T16:09:12.540479636Z caller=distributor.go:409 msg=received spanid=4b5458acd123261f traceid=3254bd41d3d557cb9f71fd3c3edb02fb
+```
+
+The important part to notice here is a `traceid` with multiple different spans, this `traceid` is the unique identifier for the entire operation of retrieving those Vets through wasmCloud. Copy that `traceid` and navigate to [http://localhost:5050](http://localhost:5050) which is where Grafana is running. Click the compass icon (explore) in the left sidebar, and then select `Tempo` in the dropdown next to **Explore**. Then you can input your trace ID in the search bar and click `Run query` in the top right. You should see a trace like this:
+![Distributed Petclinic trace](/images/petclinic_trace.png)
+
+As you can see, we retrieved the list of vets in our database in just under 40ms from the HTTP server provider, to the API actor, to the Vets actor, and finally querying the SQLDB Postgres provider. You can expand any of these spans to get more information on the specific call like the Actor ID, the operation name, the line of code that the function is on, etc.
+
+### Tracing setup for your project
+
+In order to enable tracing we'll take a look at the above example to show you what to set up. At a high level, you'll need:
+1. wasmCloud [v0.55.1](https://github.com/wasmCloud/wasmcloud-otp/releases/tag/v0.55.1) or later
+1. A tracing backend like [Grafana Tempo](https://github.com/grafana/tempo)
```

Comment:
might be worth mentioning here that tempo is in our docker-compose file because it's quick and easy, but we should probably list off a couple examples of production-grade export targets.

Reply:
Good call 👍🏻

Reply:
Mentioned Zipkin / Jaeger for open source and DataDog / Honeycomb for commercial